curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -443,6 +444,12 @@ int FileCacheManager::ReadFromS3(const std::vector<S3ReadRequest> &requests,
                     return ret;
                 }
             }
+            if (s3ClientAdaptor_->s3Metric_.get() != nullptr) {
+                s3ClientAdaptor_->s3Metric_->adaptorReadS3.bps.count << n;
```

Comment:
adaptorReadS3？ maybe read from disk cache，so this name maybe confuse others.

Reply:
fixed